package.json

```diff
@@ -7,12 +7,12 @@
     "start": "node app.js"
   },
   "engines": {
-    "node": "0.12.x"
+	  "node": "4.8.4"
```

Comment:
You might want to add caret (^) in front of the version so the people using higher versions in major 4 could use it as well.